test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildIntegrationTest.cs

```diff
@@ -25,7 +25,7 @@ public BuildIntegrationTest(BuildServerTestFixture buildServer)
         public Task Build_SimpleMvc_UsingDotnetMSBuildAndWithoutBuildServer_CanBuildSuccessfully()
             => Build_SimpleMvc_WithoutBuildServer_CanBuildSuccessfully(MSBuildProcessKind.Dotnet);
 
-        [ConditionalFact(Skip = "https://github.com/aspnet/Razor/issues/2208")]
+        [ConditionalFact]
         [OSSkipCondition(OperatingSystems.Linux)]
         [OSSkipCondition(OperatingSystems.MacOSX)]
```

Comment:
`[OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]` since you are touching this.